app/_kong_plugins/header-cert-auth/index.md

```diff
@@ -36,6 +36,144 @@ search_aliases:
   - header cert auth
   - header-cert-auth
   - authentication
+
+faqs: 
+  - q: Will the client need to encrypt the message with a private key and certificate when passing the certificate in the header?
+    a: |
+      No, the client only needs to send the target's certificate encoded in a header. {{site.base_gateway}} will validate the certificate, but it requires a high level of trust that the WAF/LB is the only entrypoint to the {{site.base_gateway}} proxy. The Header Cert Auth plugin will provide an option to secure the source, but additional layers of security are always preferable. Network level security (so that {{site.base_gateway}} only accepts requests from WAF - IP allow/deny mechanisms) and application-level security (Basic Auth or Key Auth plugins to authenticate the source first) are examples of multiple layers of security that can be applied.
+  - q: How should the certificate be passed in a client request?
```

Comment:
I'm not sure this should be hidden in an FAQ. I'd expect to see it in the  "Client certificate request" section.